usr/klibc/arch/x86_64/setjmp.S

```diff
@@ -32,7 +34,7 @@ setjmp:
 	movq %r14,40(%rdi)
 	movq %r15,48(%rdi)
 	movq %rsi,56(%rdi)		# Return address
-	ret
+	RET
```

Comment:
though REPOLINE is quite old (it's present at least on 4.x), CONFIG_RETHUNK was introduced on v5.10.133.. perhaps it's worth the `#if KERNEL_VERSION >=`.. what do you think?

Reply:
I changed my mind.. I think v.5.10 is old enough.. if anyone wants to run an older kernel, they can open an issue and we add a #ifdef.. will merge it as is ;-)